docs/GettingStarted.md

```diff
@@ -41,6 +41,7 @@ This will build the following binaries:
                 and EbpCore and NetEbpfExt drivers to be loaded.
 * sample_ebpf_ext.sys: A sample eBPF extension driver that implements a test hook (for a test program type) and test helper functions.
 * sample_ext_app.exe : A sample application for testing the sample extension driver.
+* xdp_tests.exe : Application for testing various XDP functionalities.
```

Comment:
Line 40-41 says "This requires EbpSvc service to be running, and EbpCore and NetEbpfExt drivers to be loaded." but that statement is not here.  Does that mean it has no such requirement?